pkg/app/server/grpcapi/web_api.go

```diff
@@ -1546,7 +1613,7 @@ func (a *WebAPI) GetInsightApplicationCount(ctx context.Context, req *webservice
 			return nil, status.Error(codes.NotFound, "Not found")
 		}
 		a.logger.Error("failed to load application counts", zap.Error(err))
-		return nil, status.Error(codes.Internal, "failed to load application counts")
+		return nil, gRPCEntityOperationError(err, "load application counts")
```

Comment:
could you remove those changes from this PR, since they're not related 🙏

Reply:
Right, I changed it. 🙇